dev/Generated/AnimatedVisualPlayer.properties.cpp

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
this seems to be removed by code gen for some reason.

Reply:
I see in the latest delta they're back. Any idea what happened? Did you have to do something manual?

Reply:
weird. With a new build, they got added back.